src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/ViewBuffer.cs

```diff
@@ -92,38 +92,35 @@ public int Count
         /// <inheritdoc />
         public IHtmlContentBuilder Append(string unencoded)
         {
-            if (unencoded == null)
+            if (unencoded != null)
```

Comment:
Not super familiar with this area, but what prevents the method from being inlinable here?

Reply:
Switching to a single return block; rather than two return blocks

Reply:
Just shrinking it a little to give it more chances